### PR TITLE
[BUILDING] give credits

### PR DESCRIPTION
**Pull request by MaxHillebrand**
This ready for review branch gives some more credits to projects quoted in the docs. Does anyone know of any further that should be credited?

```
- [Microsoft](https://dotnet.microsoft.com/download)
- [Qubes Operating System](https://qubes-os.org/doc/)
- [Whonix](https://whonix.org)
- [VirtualBox](https://virtualbox.org)
```

### CODE REVIEW

File: docs/building-wasabi/Credits.md [hunk 1]
```diff
@@ -8,7 +8,8 @@
 # Credits
 
 Part of the content in this documentation comes from other open source projects.
-Here are some credits, thanks and sources (in casual order):
+Here are some credits, thanks and sources (in casual order):7
```
Comment (yahiheb) [inline docs/building-wasabi/Credits.md]:
```suggestion
Here are some credits, thanks and sources (in casual order):
```

File: docs/building-wasabi/Credits.md [hunk 2]
```diff
@@ -8,7 +8,8 @@
...
+
```
Comment (yahiheb) [inline docs/building-wasabi/Credits.md]:
```suggestion
```

File: docs/building-wasabi/Credits.md [hunk 3]
```diff
@@ -19,3 +20,7 @@ Here are some credits, thanks and sources (in casual order):
 - [CryptoSeb, Cryptopaper](https://github.com/cryptoseb/CryptoPaper)
 - [Tails](https://tails.boum.org/)
 - [BitSniff](https://github.com/m417z/bitsniff/)
+- [Microsoft](https://dotnet.microsoft.com/download)
```
Comment (yahiheb) [inline docs/building-wasabi/Credits.md]: I am not sure about this one.

Comment (MaxHillebrand) [reply docs/building-wasabi/Credits.md]: We actually do not describe how to install .NET, but only link to this download and let the Microsoft documentation speak for it self, thus I think it should be included.

Comment (yahiheb) [reply docs/building-wasabi/Credits.md]: yeah but I think we shouldn't include it (specially it is just a link to download .NET stuff), plus as you said we didn't copy any information from that source.

Comment (MaxHillebrand) [reply docs/building-wasabi/Credits.md]: Ah, now I understand your point, and yes, you are right!
```suggestion
```

**Pull request merged**